build-package.sh

```diff
@@ -3,7 +3,17 @@
 # exit on any error
 set -e
 
+# pre-requisites
+npm run format
+npm test
+
+# validate branches
```

Comment:
I would personally move this above any of the other checks.

Reply:
I was actually using this to make sure that format didn't change anything...

e.g. if the `npm run format` call modifies code then the validation below will fail.